mlir/lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -449,6 +449,49 @@ static LogicalResult printOperation(CppEmitter &emitter, emitc::SubOp subOp) {
   return printBinaryOperation(emitter, operation, "-");
 }
 
+static LogicalResult emitAllExceptLast(CppEmitter &emitter,
```

Comment:
```suggestion
static LogicalResult emitSwitchCase(CppEmitter &emitter,
```
due to writing the `break;` at the end of the function, the name can be more specific.

Reply:
Agree, it looks much better.